<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer_HistoryCurve</class>
 <widget class="QWidget" name="customer_HistoryCurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="plots_groupbox_customer">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_27">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
       </property>
       <property name="text">
        <string>快速查询：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="quickSearch_customer">
       <item>
        <property name="text">
         <string>今天</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>昨天</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>最近七天</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>本月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>上个月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>自定义</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
       </property>
       <property name="text">
        <string>开始时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="startDateTimeEdit_customer"/>
     </item>
     <item>
      <widget class="QLabel" name="label_20">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
       </property>
       <property name="text">
        <string>结束时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="endDateTimeEdit_customer"/>
     </item>
     <item>
      <widget class="QPushButton" name="searchData_customer">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	width:112px;
	height:34px;
	background:rgba(233,235,237,1);
	border:1px solid rgba(112,112,112,1);
	border-radius:10px;
	font-size:17px;
	font-family:PingFang SC;
	font-weight:300;
	line-height:24px;
	color:rgba(97,97,97,1);
}</string>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportData_customer">
       <property name="styleSheet">
        <string notr="true">QPushButton {
width:112px;
height:34px;
background:rgba(81,223,0,1);
border:1px solid rgba(112,112,112,1);
border:0px;
border-radius:10px;
font-size:17px;
font-family:PingFang SC;
font-weight:300;
line-height:24px;
color:rgba(255,255,255,1);
}</string>
       </property>
       <property name="text">
        <string>数据导出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>187</width>
       <height>507</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
background:rgba(255,255,255,1);
border:1px solid rgba(191,191,191,1);
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_24">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_customer_chart_6">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>IT-02</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="picLabel_22">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_indigo.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_23">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_customer_chart_5">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>VT-02</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="picLabel_21">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_yellow.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_customer_chart_2">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>LT-02</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="picLabel_18">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_green.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_customer_chart_1">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>LT-01</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="picLabel_17">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_blue.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_customer_chart_3">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>VT-01</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="picLabel_19">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_pink.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_22">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_customer_chart_4">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>IT-01</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="picLabel_20">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_red.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
